### ISSUE TITLE
2.2.2 Pause Stop Hide

### ISSUE THREAD

**Issue by melaniephilipp** [w3c/wcag2ict#232]
Note 5: While the success criteria uses the term “information”, the WCAG 2.2 Intent section makes it clear that this is to be applied to all content. Any [content](https://www.w3.org/TR/wcag2ict-22/#content-on-and-off-the-web), whether informative or decorative, that is updated automatically, blinks, or moves may create an accessibility barrier. 

This note feels like it is adding normative implications to SC 2.2.2 that the normative SC doesn’t support.

**Comment by patrickhlauke** [comment]
While the SC wording itself uses "information", the (normative) notes for the SC talk about "content" in general. I would suggest that the use of "information" in the SC wording is merely a linguistic quirk, and not intended to mean specifically "this only applies to content that conveys information" (otherwise it would be linked to a glossary definition of "information")

**Comment by maryjom** [comment]
Hi Melanie. Thank you for taking the time to review the WCAG2ICT public draft. Our task force will review your comment and develop a response that we hope will address your concern. The response will be drafted in a comment on this issue, marked, **DRAFT RESPONSE:** until the Task Force reaches consensus.

**Comment by maryjom** [comment]
**DRAFT RESPONSE**
Note 5 is unchanged from the guidance published in the 2013 WCAG2ICT Note. At the time, it was developed in alignment with the guidance in the [Understanding SC 2.2.2 Pause, Stop, Hide Intent section](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html#intent) as well as reiterating WCAG's Note 2 for the SC which says, 
>Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the Web page (whether it is used to meet other success criteria or not) must meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).

WCAG's note uses normative ("must") as well as web-based language ("web page"), and WCAG2ICT note 5 restates or summarizes that note, and is aligned with the spirit of the Intent, without using normative or web-based language. Based on the supporting WCAG guidance, Note 5 does not seem to be expanding the scope beyond what is said in the existing WCAG note.

If you have examples of where this SC should not be applied to non-web content that should be accounted for, please provide them to help the Task Force explore this issue more. Or you can indicate that leaving Note 5 as-is is an acceptable approach.

**Comment by maryjom** [comment]
After the WCAG2ICT discussion on 5 October, we decided to open [Issue 3478](https://github.com/w3c/wcag/issues/3478) on WCAG to clarify whether the SC really means "content" when it says "information", meaning "all content on the page" is subject to this SC.  The non-normative notes say it, but not the normative text.

**Comment by melaniephilipp** [comment]
Thank you @maryjom. 

A while back, I was interested in the history of SC 2.2.2 Pause Stop Hide, and the apparent discrepancy between the use of “information” in the normative language and the broader application to “content” in the Understanding document. I dug back into the version history to see if that revealed anything about the intent of the SC. I found the following:

In the [2006/04/27 Working Draft](https://www.w3.org/TR/2006/WD-WCAG20-20060427/guidelines.html#time-limits), both parts of today's SC were scoped to "content".
In the [2007/05/17 Working Draft](https://www.w3.org/TR/2007/WD-WCAG20-20070517/#time-limits), the blinking part was scoped to "content" and the moving part was scoped to "information"
In the [2007/12/11 Working Draft](https://www.w3.org/TR/2007/WD-WCAG20-20071211/#time-limits), blinking, scrolling and updating were scoped to "information" and an additional clause was added for moving "decorative" content
In the [2008/04/30 Candidate Recommendation](https://www.w3.org/TR/2008/CR-WCAG20-20080430/#time-limits), the decorative clause was removed and only "information" remained. A note was added that [the SC was at risk](https://www.w3.org/TR/2008/CR-WCAG20-20080430/#status_risk). In that note, the "fall back" was to add "decorative content" back in "If implementation feedback from users indicates that the current version is less helpful than the version from 11 December 2007, or if it is determined that assistive technology can provide a way to stop moving content, this success criterion will revert to the Level AA fallback wording."
Decorative content was NOT added back into the [2008/11/03 Proposed Recommendation](https://www.w3.org/TR/2008/PR-WCAG20-20081103/#time-limits) doc. 

Minutes from that period of time were not really minutes, mostly a summary of resolutions. The issues referenced in bugzilla are no longer available (the issue links go to blank pages), so I can't see WHY the SC evolved from "content" to "information" and "decorative content" was added and removed.

But given the history I can see, it confirms my feeling that the SC was deliberately scoped to "information" not "content" and the Understanding doc was never updated to reflect that.

I will add this info to your issue as well.

**Comment by maryjom** [comment]
WCAG2ICT RESPONSE:

Note 5 is unchanged from the guidance published in the 2013 WCAG2ICT Note. At the time, it was developed in alignment with the guidance in the [Understanding SC 2.2.2 Pause, Stop, Hide Intent section](https://www.w3.org/WAI/WCAG22/Understanding/pause-stop-hide.html#intent) as well as reiterating WCAG's Note 2 for the SC which says,

>Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the Web page (whether it is used to meet other success criteria or not) must meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).

WCAG's note uses normative ("must") as well as web-based language ("web page"), and WCAG2ICT note 5 restates or summarizes that note, and is aligned with the spirit of the Intent, without using normative or web-based language. Based on the supporting WCAG guidance, Note 5 does not expand the scope beyond what is said in the existing WCAG note.

If you have examples of where this SC should not be applied to non-web content that should be accounted for, please provide them to help the Task Force explore this issue more. Or you can indicate that leaving Note 5 as-is is an acceptable approach.

We opened [Issue 3478](https://github.com/w3c/wcag/issues/3478) on WCAG and will watch to see how this issue is resolved.  For now, we will leave our non-normative Note 5 as-is, as it does not contradict WCAG's non-normative [Note 2](https://www.w3.org/TR/WCAG22#h-note-20). Should WCAG make an errata or otherwise clarify this SC, we will be sure to adjust our guidance so it is aligned.

**Comment by maryjom** [comment]
Closing this issue as answered.
